addons/invert-stage/addon.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "Invert Stage",
+  "description": "Inverts the stage. Useful for testing contrast and people with colorblindness.",
+  "tags": ["community"],
+  "versionAdded": "1.36.0",
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["projects", "projectEmbeds"]

```

Comment:
If this is an addon for testing projects for colorblindness, does it need to be on project embeds as well? Those are mostly used by people viewing the project, not making them.

Reply:
It could be userful for people who are colorblind themselves so I will keep this as is